### DISCUSSION TITLE
Bringing an Astro Implementation to Storybook

### DISCUSSION THREAD

**Discussion by lukemcd-huge** [storybookjs/storybook#19854] [Ideas]
[Astro](https://astro.build) is a popular web framework for building fast, content-focused websites. Astro allows you to build out static HTML files or incorporate UI frameworks such as React and Vue. If work could be done to add support for `.astro` files it would bring together two of the most powerful and talked about tools for frontend development.

**Comment by shilman** [comment]
Closing as duplicate to https://github.com/storybookjs/storybook/issues/18356, our most-upvoted issue ever!
